src/dotnet/UI.Blazor/Components/ContextMenu/BlazorContextMenu/Components/Item.razor

```diff
@@ -0,0 +1,178 @@
+    /// </summary>
+    [Parameter]
+    public string? OverrideDefaultCssClass { get; set; }
+
+    /// <summary>
+    /// Allows you to override the default css class of the <see cref="Item"/>'s li element when disabled, for full customization.
+    /// </summary>
+    [Parameter]
+    public string? OverrideDefaultDisabledCssClass { get; set; }
+
+    /// <summary>
+    /// Allows you to override the default css class of the <see cref="Item"/>'s li element when it contains a <see cref="SubMenu"/>, for full customization.
+    /// </summary>
+    [Parameter]
+    public string? OverrideDefaultCssClassWithSubMenu { get; set; }
+
+    /// <summary>
+    /// Additional css class for the <see cref="Item"/>'s li element. Use this to extend the default css.
+    /// </summary>
+    [Parameter]
+    public string? CssClass { get; set; }
+
+    /// <summary>
+    /// Additional css class for the <see cref="Item"/>'s li element when disabled. Use this to extend the default css.
+    /// </summary>
+    [Parameter]
+    public string? DisabledCssClass { get; set; }
+
+    /// <summary>
+    /// Additional css class for the <see cref="Item"/>'s li element when it contains a <see cref="SubMenu"/>. Use this to extend the default css.
+    /// </summary>
+    [Parameter]
+    public string? CssClassWithSubMenu { get; set; }
+
+    /// <summary>
+    /// The menu item's onclick handler. A <see cref="ItemClickEventArgs"/> is passed to the action.
+    /// If you want to cancel the click (i.e. stop the menu from closing), then set the "IsCanceled" event args property to "true".
+    /// </summary>
+    [Parameter]
+    public EventCallback<ItemClickEventArgs> OnClick { get; set; }
+
+    /// <summary>
+    /// Sets the item's enabled state. Default <see cref="true" />.
+    /// </summary>
+    [Parameter]
+    public bool Enabled { get; set; } = true;
+
+    /// <summary>
+    /// Sets the item's visible state. Default <see cref="true" />.
+    /// </summary>
+    [Parameter]
+    public bool Visible { get; set; } = true;
+
+    /// <summary>
+    /// A handler that can be used to set the item's <see cref="Visible"/> or <see cref="Enabled"/> status dynamically.
+    /// </summary>
+    [Parameter]
+    public EventCallback<ItemAppearingEventArgs> OnAppearing { get; set; }
+
+    /// <summary>
+    /// The id of the li element. This is optional.
+    /// </summary>
+    [Parameter]
+    public string? Id { get; set; }
+
+    /// <summary>
+    /// Allows you to override the default x position offset of the submenu (i.e. the distance of the submenu from it's parent menu).
+    /// </summary>
+    [Parameter]
+    public int? SubmenuXOffset { get; set; }
+
+    [Parameter]
+    public RenderFragment ChildContent { get; set; } = null!;
+
+
+    protected string ClassCalc
+    {
+        get
+        {
+            var template = settings.GetTemplate(Template);
+            string classes;
+            if (Enabled) {
```

Comment:
Can you explain how you manage to use 2 different coding styles - sometimes they're on the same line & sometimes - on the different ones?

Reply:
I did not reformat this part